azext_iot/tests/test_iot_central_unit.py

```diff
@@ -123,9 +123,21 @@ class Cmd:
         }
 
 
+class TestCentralDeviceAuth:
```

Comment:
Important - Add an integration test:

1) Fetch SAS group token from API (if exists), or loads from config file
2) Create key using dps-keygen
3) Provision the device using the device sdk that we use to send messages to IoT Central
4) connect the device
5) Send a message to central, check if it got there using `monitor-events`, or check that the device_state is "registered" or something